### DISCUSSION TITLE
最強的伺服器作業系統

### DISCUSSION THREAD

**Discussion by 32844583** [1101-GS3073-C/Python-Community-GS3073-C#543] [Ideas]
### Linux 
Linux 作業系統在伺服器領域中的使用越來越廣泛，可以用於構建 WWW 伺服器、資料庫伺服器、負載均衡伺服器、郵件伺服器、DNS 伺服器、代理伺服器、路由器等，使企業降低運營成本，同時也獲得了 Linux 系統帶來的高穩定性和高可靠性。近年來，Linux 已經普及到電信、金融、政府、教育、銀行、石油等各個行業，並得到各大硬體廠商的支援。預計 Linux 在伺服器市場的前景是光明的。

教學:https://www.youtube.com/watch?v=-fzO7iWCSWY

**Comment by BingRong1217** [comment]
好想學會Linux!!!

**Comment by awkward-willy** [comment]
Linux 作為一個開源的作業系統，的確有許許多多的好處，漏洞也相對更少，學會如何操作會是一大優勢之一，未來老師也有開相關課程，或許是入門的不錯選擇。

**Comment by Ben6uo** [comment]
但Linux是一個比較高階 感覺很難

**Comment by williamyen0623** [comment]
Linux大部分功能都要學會如何使用終端機，對於習慣使用圖形化介面的我光是看到就頭昏眼花，希望下學期修家凱老師的Linux邊緣運算可以把Linux學好！

**Comment by HsuChen88** [comment]
Linux算是易學難精嗎，學過一點，但目前還不知道如何利用他的優點。
